app/models/tournament.rb

```diff
@@ -1,4 +1,5 @@
 class Tournament < ApplicationRecord
+  require 'csv'
```

Comment:
Since we're only using csv for this one function, I require it here rather than always loading it via application.rb